railties/lib/rails/commands/db/system/change/change_command.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "rails/generators"
+require "rails/generators/rails/db/system/change/change_generator"
+
+module Rails
+  module Command
+    module Db
+      module System
+        class ChangeCommand < Base # :nodoc:
+          class_option :to, desc: "The DBMS to switch to."
```

Comment:
Do we generally say DBMS elsewhere?

Reply:
Try "database type"? It's not brilliant, but maybe an improvement? Or "database platform"?

Reply:
Using "database system" would match the command name and reinforce the concept. But it's perhaps not expounding on what a "database system" means, that meaning only comes from seeing MySQL, Postgres and the gang.

Reply:
Agree that `database system` makes the most sense given the command name.